diff --git a/solana/programs/bridge/src/lib.rs b/solana/programs/bridge/src/lib.rs
index 70cb385..fbdfde3 100644
--- a/solana/programs/bridge/src/lib.rs
+++ b/solana/programs/bridge/src/lib.rs
@@ -249,6 +249,7 @@ pub mod bridge {
         amount: u64,
         call: Option<Call>,
     ) -> Result<()> {
+        msg!("UPGRADED");
         bridge_sol_handler(ctx, outgoing_message_salt, to, amount, call)
     }
 
