diff --git a/interfaces/L2/IFeeVault.sol b/interfaces/L2/IFeeVault.sol
--- a/interfaces/L2/IFeeVault.sol
+++ b/interfaces/L2/IFeeVault.sol
@@ -32,7 +32,7 @@ interface IFeeVault {
     function minWithdrawalAmount() external view returns (uint256);
     function recipient() external view returns (address);
     function totalProcessed() external view returns (uint256);
-    function withdraw() external returns (uint256 value_);
+    function withdraw() external;
     function withdrawalNetwork() external view returns (Types.WithdrawalNetwork);
     function setMinWithdrawalAmount(uint256 _newMinWithdrawalAmount) external;
     function setRecipient(address _newRecipient) external;
